autometa/common/exceptions.py

```diff
@@ -150,7 +113,16 @@ def __str__(self):
         str
             Message written alongside raising the exception
         """
-        return self.value
+        message = """
+
+        NCBI databases nodes.dmp, names.dmp and merged.dmp are out of sync with each other.
+
+        To download the databses visit: ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdump.tar.gz
+
+        Required files within taxdump tarball are *nodes.dmp*, *names.dmp* and *merged.dmp*
+
+        """
+        return message
```

Comment:
Does this overwrite/not emit the original message that was passed to `DatabaseOutOfSyncError`?

Reply:
Yes it will. The original message will be logged while this warning would be emitted.

Reply:
This seems like we are obfuscating the issue rather than helping the user further. Why would we not want to emit the message that is passed to the exception?